### ISSUE TITLE
Plans to put into pip?

### ISSUE THREAD

**Issue by seanorama** [mantl/terraform.py#71]
Are there plans to make available in pip?

**Comment by thomasvincent** [comment]
yes, a couple of projects have borrowed this script. Putting it in PIP so we are not duplicating resources is not a bad idea.

**Comment by seanorama** [comment]
@thomasvincent I've been installing with the following but obviously this is not the best way:

```
pip install git+https://github.com/mantl/terraform.py.git
```

**Comment by Theaxiom** [comment]
Thanks for the suggestion @seanorama !
